### ISSUE TITLE
Please allow other spinners on Windows

### ISSUE THREAD

**Issue by ffMathy** [sindresorhus/ora#159]
Often, CLIs etc are used via VS Code's terminal, or some other terminal that does have support. For instance, Windows Terminal.

**Comment by sindresorhus** [comment]
Sure, but we need to detect those terminals that do support it.

**Comment by ffMathy** [comment]
Gotcha - that makes sense.

**Comment by woacademy** [comment]
@sindresorhus could you elaborate on https://github.com/sindresorhus/figures/pull/27#issuecomment-504764201?

I'm aware of the open issue you linked, however, `WT_SESSION` was added for this reason - https://github.com/microsoft/terminal/pull/897. Could we use that until the other issue is closed out at some (possible distant) point in the future?

**Comment by dsl101** [comment]
Or, just allow `unicode: true` in the options object, to force use of unicode. Caveat emptor, and all that? I created a PR which enables this on windows (in an appropriate terminal) to work with all spinners. Really only useful for local scripts, not those intended to be published though.

**Comment by dsl101** [comment]
Does [this](https://github.com/jcarpanelli/spinnies/blob/90e33ee6384eab6d4783a506301ab3dbaee24b00/utils.js#L99) meet your requirements? I'd be happy to alter the PR... It looks like that issue you linked to is going to be around for a while.

**Comment by sindresorhus** [comment]
Yes, I'm ok with that, since it doesn't look like Windows Terminal will have proper detection anytime soon.
